manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -19,11 +23,11 @@ rules:
   # istio configuration
   # removing CRD permissions can break older versions of Istio running alongside this control plane (https://github.com/istio/istio/issues/29382)
   # please proceed with caution
-  - apiGroups: ["config.istio.io", "security.istio.io", "networking.istio.io", "authentication.istio.io", "rbac.istio.io", "telemetry.istio.io"]
+  - apiGroups: ["config.istio.io", "security.istio.io", "networking.istio.io", "authentication.istio.io", "rbac.istio.io"{{- if not .Values.global.externalIstiod }}, "telemetry.istio.io"{{- end }}]
```

Comment:
what does telemetry have to do with external istiod?

Reply:
Telemetry is not installed on the remote cluster, so this is only including the telemetry api group if it is NOT using an external istiod. Again, this was the difference between the discovery and istiod-remote version of the chart.

Reply:
What do you mean its not installed? I don't see how Telemetry is any different from VirtualService or any other CRD

Reply:
If we are just trying to preserve the exact same behavior as before - lets not propogate bugs with complex `if` statements, I think it would be better to just fix it to be consistent and always have telemetry

Reply:
> What do you mean its not installed?

I think my confusion stems from the fact that there is also a version of this ClusterRole in the base charts, which is also installed on config clusters (and does have the telemetry api group). I don't really understand when this one will be used and when the base one is used. I had assumed that this one is only on non-config-clusters where telemetry isn't installed, but thinking about it, that would lead to the question of why other apiGroups (e.g., networking) are needed either. I was trying to generate the exact same manifest as the istiod-remote chart currently, but if it's a bug, I agree it makes no sense to propagate.

One question, though. What are the `telemetry.istio.io` APIs? I couldn't fine anything in the docs.